docs/customization.md

```diff
@@ -0,0 +1,55 @@
+# Customizing Vivify
+
+Vivify offers various configuration options. It aims to have sensible defaults
+while being built for maximal customizability.
+
+Vivify will look for an optional config file at `~/.vivify/config.json` and
+`~/.vivify.json`. This file should contain a JSON object that can have the
+following optional keys:
+
+- **`"styles"`**\
+  A path to a single custom style sheet, or an array of multiple style sheets
+  applied in order. These will be applied after Vivify's [default
+  styles](./static/) are applied so that there are always sensible fallbacks but
+  you can override everything.
+- **`"scripts"`**\
+  A path to a single custom JavaScript to inject into the viewing pages, or an
+  array of multiple custom scripts.
+- **`"dirListIgnore"`**\
+  A path to a file with globs to ignore in Vivify's directory viewer, or an
+  array of multiple paths to ignore files. The syntax here is the same as in
+  `.gitignore` files.
+- **`"port"`**\
+  The port Vivify's server should run on; this will be overwritten by
+  the environment variable `VIV_PORT` (default is 31622)
+- **`"timeout"`**\
+  How long the server should wait in milliseconds before shutting down after the
+  last client disconnected; this will be overwritten by the environment variable
+  `VIV_TIMEOUT` (default is 10000)
+- **`"katexOptions"`**\
+  [Available KaTeX options](https://katex.org/docs/options.html), such as
+
+  ```json
+  {
+    "errorColor": "#cc0000",
+    "macros": {
+      "\\RR": "\\mathbb{R}"
+    }
+  }
+  ```
+
+- **`"pageTitle"`**\
+  JavaScript code that will be evaluated to determine the viewer's page title.
+  Here, the variable `components` is set to a string array of path components
+  for the current file, e.g. `['/', 'Users', 'you', 'file.txt']`. If this
+  evaluation fails, the title will be *custom title error* and you will see the
+  error message on the page. The default title are the last two components
+  joined with the path separator, e.g.  `you/file.txt`
```

Comment:
The `you/file.txt` example feels like it's technically out of date since now it'd be `~/file.txt` :D

Perhaps super nitpicky

But also I feel like this option is hard to explain and the extra confusion with that would be worth avoiding